src/stories/Library/Modals/modal-find-on-shelf/modal-find-on-shelf.scss

```diff
@@ -0,0 +1,30 @@
+.modal-find-on-shelf {
+  padding: 0;
+  overflow-y: scroll;
+  align-items: flex-start;
+
+  @include breakpoint-m {
+    padding: 0 215px $s-4xl 215px;
+  }
+
+  &__headline {
+    margin: 72px $s-md 0 $s-md;
+
+    @include breakpoint-m {
+      margin-top: $s-4xl;
+    }
+  }
+
+  &__caption {
+    margin: $s-2xl 0 $s-md $s-md;
+
+    @include breakpoint-m {
+      margin: 96px 0 $s-sm 16px;
+    }
+  }
+
+  .disclosure {
+    width: 100%;
+    margin: 0;
+  }
```

Comment:
Please reconsider the BEM structure.

If we want different styling for the `.disclosure` then it should be a modifier for that block instead of nesting different blocks.

Reply:
That makes total sense. Thanks :)